diff -rupN Python-3.8.1.old/configure Python-3.8.1.new/configure
--- Python-3.8.1.old/configure	2019-12-18 12:21:23.000000000 -0500
+++ Python-3.8.1.new/configure	2020-02-10 15:14:33.000000000 -0500
@@ -5297,7 +5297,7 @@ cat >> conftest.c <<EOF
 EOF
 
 if $CPP $CPPFLAGS conftest.c >conftest.out 2>/dev/null; then
-  PLATFORM_TRIPLET=`grep -v '^#' conftest.out | grep -v '^ *$' | tr -d ' 	'`
+  PLATFORM_TRIPLET=`grep -v '^#' conftest.out | grep -v '^ *$' | grep -v '__asm' | tr -d ' 	'`
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: $PLATFORM_TRIPLET" >&5
 $as_echo "$PLATFORM_TRIPLET" >&6; }
 else
@@ -11781,38 +11781,38 @@ $as_echo "no" >&6; }
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for memfd_create" >&5
-$as_echo_n "checking for memfd_create... " >&6; }
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-#ifdef HAVE_SYS_MMAN_H
-#include <sys/mman.h>
-#endif
-#ifdef HAVE_SYS_MEMFD_H
-#include <sys/memfd.h>
-#endif
-
-int
-main ()
-{
-void *x=memfd_create
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_compile "$LINENO"; then :
-
-$as_echo "#define HAVE_MEMFD_CREATE 1" >>confdefs.h
-
-   { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-else
-  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-
-fi
-rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
+# { $as_echo "$as_me:${as_lineno-$LINENO}: checking for memfd_create" >&5
+# $as_echo_n "checking for memfd_create... " >&6; }
+# cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+# /* end confdefs.h.  */
+# 
+# #ifdef HAVE_SYS_MMAN_H
+# #include <sys/mman.h>
+# #endif
+# #ifdef HAVE_SYS_MEMFD_H
+# #include <sys/memfd.h>
+# #endif
+# 
+# int
+# main ()
+# {
+# void *x=memfd_create
+#   ;
+#   return 0;
+# }
+# _ACEOF
+# if ac_fn_c_try_compile "$LINENO"; then :
+# 
+# $as_echo "#define HAVE_MEMFD_CREATE 1" >>confdefs.h
+# 
+#    { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
+# $as_echo "yes" >&6; }
+# else
+#   { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
+# $as_echo "no" >&6; }
+# 
+# fi
+# rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 
 # On some systems (eg. FreeBSD 5), we would find a definition of the
 # functions ctermid_r, setgroups in the library, but no prototype
diff -rupN Python-3.8.1.old/configure.ac Python-3.8.1.new/configure.ac
--- Python-3.8.1.old/configure.ac	2019-12-18 12:21:23.000000000 -0500
+++ Python-3.8.1.new/configure.ac	2020-02-10 15:15:27.000000000 -0500
@@ -851,7 +851,7 @@ cat >> conftest.c <<EOF
 EOF
 
 if $CPP $CPPFLAGS conftest.c >conftest.out 2>/dev/null; then
-  PLATFORM_TRIPLET=`grep -v '^#' conftest.out | grep -v '^ *$' | tr -d ' 	'`
+  PLATFORM_TRIPLET=`grep -v '^#' conftest.out | grep -v '^ *$' | grep -v '__asm' | tr -d ' 	'`
   AC_MSG_RESULT([$PLATFORM_TRIPLET])
 else
   AC_MSG_RESULT([none])
@@ -3646,19 +3646,19 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
   [AC_MSG_RESULT(no)
 ])
 
-AC_MSG_CHECKING(for memfd_create)
-AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
-#ifdef HAVE_SYS_MMAN_H
-#include <sys/mman.h>
-#endif
-#ifdef HAVE_SYS_MEMFD_H
-#include <sys/memfd.h>
-#endif
-]], [[void *x=memfd_create]])],
-  [AC_DEFINE(HAVE_MEMFD_CREATE, 1, Define if you have the 'memfd_create' function.)
-   AC_MSG_RESULT(yes)],
-  [AC_MSG_RESULT(no)
-])
+# AC_MSG_CHECKING(for memfd_create)
+# AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
+# #ifdef HAVE_SYS_MMAN_H
+# #include <sys/mman.h>
+# #endif
+# #ifdef HAVE_SYS_MEMFD_H
+# #include <sys/memfd.h>
+# #endif
+# ]], [[void *x=memfd_create]])],
+#   [AC_DEFINE(HAVE_MEMFD_CREATE, 1, Define if you have the 'memfd_create' function.)
+#    AC_MSG_RESULT(yes)],
+#   [AC_MSG_RESULT(no)
+# ])
 
 # On some systems (eg. FreeBSD 5), we would find a definition of the
 # functions ctermid_r, setgroups in the library, but no prototype
